examples/rhoai-quickstart/feast-demo-quickstart.ipynb

```diff
@@ -0,0 +1,1530 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2f6dd4fd-b018-43c2-8467-b2f4d3a0994d",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
+    "tags": []
+   },
+   "source": [
+    "### Starting feast online sever"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "a3243672-4877-4588-b628-4d344bb55c7c",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/opt/app-root/lib64/python3.9/site-packages/uvicorn/workers.py:16: DeprecationWarning: The `uvicorn.workers` module is deprecated. Please use `uvicorn-worker` package instead.\n",
+      "For more details, see https://github.com/Kludex/uvicorn-worker.\n",
+      "  warnings.warn(\n",
+      "[2024-09-24 03:09:58 +0000] [11973] [INFO] Starting gunicorn 23.0.0\n",
+      "[2024-09-24 03:09:58 +0000] [11973] [INFO] Listening at: http://127.0.0.1:6566 (11973)\n",
+      "[2024-09-24 03:09:58 +0000] [11973] [INFO] Using worker: uvicorn.workers.UvicornWorker\n",
+      "[2024-09-24 03:09:58 +0000] [12007] [INFO] Booting worker with pid: 12007\n",
+      "[2024-09-24 03:09:58 +0000] [12007] [INFO] Started server process [12007]\n",
+      "[2024-09-24 03:09:58 +0000] [12007] [INFO] Waiting for application startup.\n",
+      "[2024-09-24 03:09:58 +0000] [12007] [INFO] Application startup complete.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import subprocess\n",
+    "\n",
+    "# Run feast serve in the background\n",
+    "feast_online_server_process = subprocess.Popen([\"feast\", \"serve\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "cde4756b-ac6a-490b-bbcc-89d9b1ba3285",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1001130+   11973   11692 51 03:09 ?        00:00:04 /opt/app-root/bin/python3.9 /opt/app-root/bin/feast serve\n",
+      "1001130+   12007   11973  0 03:09 ?        00:00:00 /opt/app-root/bin/python3.9 /opt/app-root/bin/feast serve\n",
+      "1001130+   12014   12011  0 03:10 ?        00:00:00 grep feast serve\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%sh\n",
+    "# checking if the online server process started.\n",
+    "ps -ef | grep 'feast serve'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee01d7e9-90cf-405e-8f0b-bcf648f45d52",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
+    "tags": []
+   },
+   "source": [
+    "### Retrieving the features using online remote client "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "fbd81a9e-a388-4a1b-a2a0-3f1f838194d7",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "remote-online feature_Store.yaml file has been created.\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
if the file content is fixed, you can also keep it in the git repo as is, there's no need to generate it from the notebook

Reply:
I did that way initially, later swiched it to the code so that when we look at the example it is clear to go through notebook. We are generating the other files using `feast init` so thought of generating rest of the files using code to keep it consistent. Please let me know if you think otherwise.